### ISSUE TITLE
502 Gateway error HTTPS SSL: error:1408F10B:SSL wrong version

### ISSUE THREAD

**Issue by frisodubach** [jc21/nginx-proxy-manager#1160]
I get a 502 Gateway error no matter what, when using https proxy host for Nextcloud and Ghost, using DNS provider Cloudflare. 

**Nginx Proxy Manager Version**
2.8.1

**To Reproduce**
1. Go to 'Proxy Hosts'
2. Click on 'Edit'
3. Change scheme to 'https'
4. See error 502 bad gateway

**Operating System**

Raspberry PiOS

**Additional context**

Found the SSL error in /data/logs/error.log

**Comment by TheCatster** [comment]
I have the same exact issue, with the same error (only difference being I run NPM in docker on Debian x86_64.) Did you ever find a solution to this?

**Comment by frisodubach** [comment]
No sadly not. I haven't had time to work on figuring out the issue. Good to hear that it's not an issue with just my architecture.
I run NPM in docker too btw.

**Comment by TheCatster** [comment]
Yeah, some of the services I'm running are mission critical, so I just resorted to switching to Caddy. This was a fun project to play with.

**Comment by inthebrilliantblue** [comment]
So Im thinking this issue is being caused by SNI. If you are going -> nginxproxymanager HTTPS -> internal server HTTPS with SNI not required, it works. But if you are requiring SNI (such as on IIS), you will get the 502 error.
